src/instructlab/model/train.py

```diff
@@ -222,7 +224,9 @@ def get_files(pattern):
                 "Found multiple files from `ilab generate`. Using the most recent generation.",
                 fg="yellow",
             )
-        # First file is latest (by above reverse sort and timestamped names)
+        # The first file is latest
+        ctx.obj.logger.debug("train_file=%s", train_files[0])
+        ctx.obj.logger.debug("test_file=%s", test_files[0])
```

Comment:
This will need to be changed to use the new logging style where each module uses it's own logger.

```suggestion
        # The first file is latest
        logger.debug("train_file=%s", train_files[0])
        logger.debug("test_file=%s", test_files[0])
```

Near the top of the file

```python
import logging

logger = logging.getLogger(__name__)